LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# The MIT License (MIT)
+
+Copyright (c) 2015 Your Name
```

Comment:
The Pull Request for Boilerplate for that is over at  https://github.com/jstransformers/boilerplate/pull/19 .